features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -280,3 +280,47 @@ Scenario: Supplier asks a clarification question
   And I click 'Ask question'
   Then I see a success banner message containing 'Your question has been sent.'
 
+@sign-framework-agreement-call-to-action
+Scenario: Supplier can see sign framework agreement call to action
+  Given I have a live digital outcomes and specialists framework
+  And that supplier is on that framework
+  And that supplier has a service on the digital-specialists lot
+  And I have a live digital-specialists brief
+  And that supplier has filled in their application but not submitted it
```

Comment:
I see this step already exists merged in, but I'd **really** like to see it renamed to be clear that it is to do with an application for an opportunity, and not some other sort of application. Something like "that supplier has filled in their response to that brief but not submitted it".